util/systemd/systemd_linux.go

```diff
@@ -18,6 +18,25 @@ var getNotifyOnce struct {
 	v *sdnotify.Notifier
 }
 
+type logOnce struct {
+	sync.Once
+}
+
+func (l *logOnce) log(format string, args ...interface{}) {
+	l.Once.Do(func() {
+		log.Printf(format, args...)
+	})
+}
+
+var (
+	readyOnce = &logOnce{
+		Once: sync.Once{},
```

Comment:
No need to initialize these. The zero value is usable.

Reply:
Ah, sorry, rust habits kicked in.